src/components/InvalidAssignment/InvalidAssignment.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../pages/Index/index.scss";
+
+.InvalidAssignment {
+  background-color: #b1b0ff;
+  width: 100vw;
+  height: 7vw;
+  border-radius: 1em;
+  border: 2px solid black;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  margin: auto;
+}
+
+.Message {
+  padding-top: 1vw;
+  width: 90vw;
+  height: 4vw;
+  font-family: "Roboto", sans-serif;
+  font-size: 1.5em;
+  text-align: center;
+  margin: auto;
+  display: inline-block;
+}
+
+.CloseMessage {
+  color: $close-button-color;
+  background-color: $assignments-background-color;
+  font-family: "Roboto", sans-serif;
+  font-size: 1.5em;
+  width: 5vw;
+  border-radius: 8px;
```

Comment:
same thing about the borders here, ever since that whole two button styles thing I'd rather have buttons look like they do on the CreateAssignmentPage

Reply:
wait I think this just makes the button rounded; was there anything about the look of the button you'd want me to change though (based on pic above)?